libraries/provider_rack_user.rb

```diff
@@ -1,4 +1,5 @@
 require 'chef/resource/lwrp_base'
+require 'shadow'
```

Comment:
Valid point, but does that mean that anything that is not a default gem, but included in chef, should be installed by chef?